post-scan-actions/aws-nodejs-securityhub-integration/lambda/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "securityhub-integration",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "zip": "bestzip bundle.zip *",
+    "package": "npm install && npm run zip"
```

The readme say `npm run build`
```suggestion
    "build": "npm install && npm run zip"
```

---

Well I'm okay with both. It's just your readme states `build` but the code is `package`.